setup.cfg

```diff
@@ -7,7 +7,9 @@ message = Bump up to version {new_version}.
 [flake8]
 filename = *.py,*.pyx,*.pxi,*.pxd
 exclude = .eggs,*.egg,build,extern,doc/source/examples
-ignore = E901,E225,E226,E227,E999,W503,W504,F401
+ignore = E225,E226,E227,E999,W504
```

Comment:
Is there a good reason to ignore these? I'm generally in favor of enforcing white space around operators.

Reply:
Yes - `flake8` doesn't naturally support Cython syntax. `flake8` reports a lot of false positives in `pyx` and `pxd` files if we don't ignore this subset of errors/warnings. However, there were a few errors/warnings that I had originally ignored for other reasons (I didn't know enough at the time to resolve them, but they're fixed now).

Reply:
(To build this list, I disabled all the flags and re-enabled the flags one at a time to check what each one did. That way I know this list is minimal, while rejecting the false positives that `flake8` finds at its default settings.)

Reply:
Ah right, it thinks pointers are multiplication. That makes sense.